draft-ietf-quic-recovery.md

```diff
@@ -1258,6 +1255,12 @@ kPersistentCongestionThreshold:
 Variables required to implement the congestion control mechanisms
 are described in this section.
 
+max_datagram_size:
+: The sender's current maximum payload size. Does not include UDP or IP
+  overhead.  The max datagram size is used for calculating initial and
+  minimum congestion windows. The RECOMMENDED value is 1200 bytes, may be
```

Comment:
I think this is actually wrong. (i) It's used for computing cwnd increases as well, but I would drop that list, and (ii) we don't need to recommend 1200, we need to require at least 1200, in keeping with the transport draft.

```suggestion
The sender's current maximum payload size. Does not include UDP or IP overhead.  The max datagram size is used for calculating  minimum congestion windows. An endpoint sets the value of this variable based on its PMTU, with a minimum value of 1200 bytes (see Section 14 of {{QUIC-TRANSPORT}}).
```